frontend/packages/dev-console/src/components/pipelines/pipeline-topology/InvalidTaskListNode.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { Node, observer } from '@console/topology';
+import { BUILDER_NODE_ERROR_RADIUS } from './const';
+import ErrorNodeDecorator from './ErrorNodeDecorator';
+import TaskListNode from './TaskListNode';
+import { TaskListNodeModelData } from './types';
+
+const InvalidTaskListNode: React.FC<{ element: Node }> = ({ element }) => {
+  const data: TaskListNodeModelData = element.getData();
```

Comment:
It's a little bit opinionated, but you can use a generic type here which defines the getData type.

```diff
- const InvalidTaskListNode: React.FC<{ element: Node }> = ({ element }) => {
-   const data: TaskListNodeModelData = element.getData();
+ const InvalidTaskListNode: React.FC<{ element: Node<NodeModel, TaskListNodeModelData> }> = ({ element }) => {
+   const data = element.getData();
```

Would require some reformatting. What do you thing? Maybe you do not need the data assignment in this case.

Reply:
Nice suggestion! Changed this one and all the other nodes too.